scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -165,47 +227,23 @@ function doSetup() {
 
 # $1: plugin config properties file
 function doAddPluginFromConfig() {
-  if [ -f "${1}" ]
+  local plugin_properties_file="${1}"
+  local optional_Plugin_Active="${2}"
+  if [ -f "${plugin_properties_file}" ]
   then
-    doLoadPluginProperties "${1}"
-    local url
-    if [ -n "${plugin_mirror_url}" ]
+    doLoadPluginProperties "${plugin_properties_file}"
+    if [ -n "${plugin_id}" ] && [ -n "$plugin_url" ] && [ "$plugin_active" = "true" ] || [ "${optional_Plugin_Active}" = "true" ] 
     then
-      url="${plugin_mirror_url}"
```

Comment:
you broke the `plugin_mirror_url` feature with your refactoring.

Reply:
Readded the code for the plugin_mirror_url feature.